hrm-domain/hrm-core/case/caseSection/sql/readSql.ts

```diff
@@ -38,27 +38,29 @@ export const SELECT_CASE_SECTION_BY_ID = `
     AND "sectionId" = $<sectionId>
     `;
 
-export const SELECT_CASE_SECTIONS_FOR_TIMELINE = `
+const selectSectionsForTimeline = (filterByType: boolean) => `
```

Comment:
Rather than using `filterByType: boolean`, couldn't we just pass down the `sectionTypes` object to make explicit what is the dependency that depends on the filtering? Or is there a reason why that's not possible? I'm just asking cause it took me some time to understand what this `filterByType` flag was for.